src/domain/events/fields.tsx

```diff
@@ -22,7 +22,3 @@ export const PublishedField = ({
     </Typography>
   );
 };
-
-PublishedField.defaultProps = {
-  label: 'events.fields.publishedAt.label',
-};
```

Comment:
Does removing this affect functionality? If it does, should the functionality be added back in some other way? Or is the functionality not needed?